src/android/Zoom.java

```diff
@@ -236,26 +241,31 @@ private void login(String username, String password, CallbackContext callbackCon
         }
 
         // bind listener.
-        this.mZoomSDK.addAuthenticationListener(this);
+        mZoomSDK.addAuthenticationListener(this);
         // Try to log user in.
 
         //exec() call to Zoom.login blocked the main thread for 28ms. Plugin should use CordovaInterface.getThreadPool().
         cordova.getThreadPool().execute(new Runnable() {
            @Override
            public void run() {
                ZoomSDK mZoomSDK = ZoomSDK.getInstance();
-
-               int response = mZoomSDK.loginWithZoom(username, password);
-               if (DEBUG) {
-                   Log.v(TAG, "[Login response ^^^^^^^^^^^^^^]= " + response);
-               }
-
-               if (response != ZoomApiError.ZOOM_API_ERROR_SUCCESS) {
-                   PluginResult pluginResult = new PluginResult(PluginResult.Status.ERROR,
-                           getApiErrorMessage(response));
-                   pluginResult.setKeepCallback(true);
-                   callbackContext.sendPluginResult(pluginResult);
-               }
+                // Note : As of v5.9.0, the SDK no longer supports directly logging in.
```

Comment:
Login will now be using OAuth. We are not utilising login flow in our app thus commenting it out.